packages/babel-core/test/_transformation-helper.js

```diff
@@ -107,8 +107,8 @@ function run(task, done) {
         }
       };
 
-      var fn = new Function("require", "done", "exports", execCode);
```

Comment:
this was a bug. Done looks like it's never used because it currently refers to `assert`